articles/quickstart.md

```diff
@@ -8,11 +8,17 @@ ms.topic: tutorial
 uid: microsoft.quantum.write-program
 ---
 
-# Quantum basics with Q#
+# Quickstart: Exploring entanglement with Q\#
```

Comment:
1- The should change from Exploring to Explore
2 - This should be moving to the Tutorials category, so the title (and references to it) should be "Tutorial: Explore entanglement with Q#". The TOC does not use the "Tutorials" prefix.

Reply:
Done